### PR TITLE
Release/2.19.0

### PR DESCRIPTION
**Pull request by eronisko**
### Fixed
- paging issues with user collections
- Enable adding to user collections in collection view

### Changed
- autocomplete for item-author and item-authority-role fiels
- ckeditor allows to add slick

### CODE REVIEW

File: webpack.mix.js [hunk 1]
```diff
@@ -15,9 +15,12 @@ if (process.env.NODE_ENV == 'testing') {
     mix.options({ processCssUrls: false });
 }
 
+
+
 mix
     // JavaScript
     .js('resources/js/app.js', 'public/js')
+    .js('resources/js/admin.js', 'public/js')
```
Comment (igor-kamil) [inline webpack.mix.js:23]: so `public/js/admin.js` should be gitignored (now it's not)

**Review by igor-kamil — COMMENTED** [review]
looks fine to me. just see my comment

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
 
 ## [Unreleased]
+## [2.19.0] - 2021-02-18
+### Fixed
+- paging issues with user collections
+- Enable adding to user collections in collection view
+
+### Changed
+- autocomplete for item-author and item-authority-role fiels
+- ckeditor allows to add slick
```
Comment (igor-kamil) [inline CHANGELOG.md:12]:
```suggestion
- ckeditor allows to add slick
- karasek importer
```

File: webpack.mix.js (hunk 1 above)
Comment (eronisko) [reply webpack.mix.js:23]: It is in #449 so I've left it as-is in here

**Review by igor-kamil — APPROVED** [review]
OK

**Pull request merged**